source/iterator/cdc_iterator.go

```diff
@@ -0,0 +1,263 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package iterator
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sort"
+	"strconv"
+	"time"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	marketoclient "github.com/rustiever/conduit-connector-marketo/marketo-client"
+	"github.com/rustiever/conduit-connector-marketo/source/position"
+	"gopkg.in/tomb.v2"
+)
+
+const (
+	ActivityTypeIDNewLead         = 12
+	ActivityTypeIDChangeDataValue = 13
```

Comment:
Why 12 and 13?

Reply:
`ActivityTypeID` for capturing CDC events of **New Lead** and **Updated Lead**. these Id's are used to filter and get only required CDC events.

Reply:
So these values are taken from the Marketo API? If so, let's please have a link to the doc mentioning those values.

Reply:
Actually i got these values from blog written by Marketo Team. Is it okay i add that link? 
Currently i added API link which gives all types of ActivityTypeIDs.

Reply:
Whatever link provides an explanation for where we have taken the values from is good.: )

Reply:
ok then, ill add the blog link as well